### PR TITLE
added component rendering and behavior event shell for components

### PR DESCRIPTION
**Pull request by jesus-santamarca**
Added complete component nesting for functional content layout components. Only applied for Color Palette Component, no API used, only object manipulation for base behavior.

### CODE REVIEW

File: src/components/ContentComp/ColorPalette/ColorPallete_CardArray.vue [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        </color-palette-card>
+    </div>
+</template>
+
+<script>
+import colorPaletteCard from './templates/colorPalette_card'
+import colorPaletteCardEdit from './templates/colorPalette_card_edit'
+/* import colorPaletteSample from './templates/colorPalette_sample'
+import colorPaletteCircle from './templates/colorPalette_circle'
+import colorPaletteTable from './templates/colorPalette_table' */
```
Comment (pedroxido) [inline src/components/ContentComp/ColorPalette/ColorPallete_CardArray.vue]: vas a usar esto a futuro? o por que esta comentado :o

Comment (jesus-santamarca) [reply src/components/ContentComp/ColorPalette/ColorPallete_CardArray.vue]: Son los imports de los item components. Los deje comentados para terminar de añadir layouts en los otros arrays de muestras de color.

File: src/components/ContentComp/ColorePalette_c.vue [hunk 2]
```diff
@@ -57,39 +57,243 @@
...
+          cancelButtonClass: 'md-button md-danger',
+          buttonsStyling: false
+        }).then(() => {
+          Swal.fire({
+            type: 'success',
+            html: 'You entered: <strong>' + $('#md-input').val() + '</strong>',
+            confirmButtonClass: 'md-button md-success',
+            buttonsStyling: false
+          })
+        })
```
Comment (pedroxido) [inline src/components/ContentComp/ColorePalette_c.vue]: this should be refactored soon 👍

Comment (pedroxido) [reply src/components/ContentComp/ColorePalette_c.vue]: not now :P

File: src/plugins/vuetify.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import Vue from 'vue'
```
Comment (pedroxido) [inline src/plugins/vuetify.js]: don't forget to remove this file if not used

**Pull request merged**